src/utilities/fe_mesh.jl

```diff
@@ -0,0 +1,162 @@
+# Typ definitions
+type FEBoundary # Tänkte typ parametrisera detta på nåt sätt så att man har FEboundary i 2D, 1D, och 0D
+    dim::Int
+    topology::Array{Int}
+end
+
+type FEMesh{dim, T <: Real}
+    coords::Vector{Vec{dim,T}} # Node coordinates
+    topology::Matrix{Int} # Element nodes
+    boundary::Vector{Vector{}} # Boundary nodes
+end
+
+
+function FEMesh(dim)
```

Comment:
In my view is a function space a completely unrelated to a mesh. You can have mesh for video game models for example. I think it is fine to just give the dimension and shape etc as separate input arguments.